src/components/src/common/tippy-tooltip.tsx

```diff
@@ -5,7 +5,7 @@ import Tippy, {TippyProps} from '@tippyjs/react';
 import React, {useCallback, useRef, useState} from 'react';
 import styled from 'styled-components';
 
-import {RootContext} from '../';
+import {RootContext} from '..';
```

Comment:
This actually looks quite bad. I don't recommend importing from a directory, much less from an enclosing directory, because implicitly one would assume that is a circular import..